Library/Styles/BaseStyles.swift

```diff
@@ -81,6 +81,12 @@ public func dropShadowStyle <V: UIViewProtocol> (radius: CGFloat = 2.0,
       <> V.lens.layer.shadowOffset .~ offset
 }
 
+public func rewardDropShadowStyle <V: UIViewProtocol> () -> ((V) -> V) {
+  return dropShadowStyle(radius: 5.0, offset: .init(width: 0, height: 2.0))
+    <> V.lens.layer.shadowOpacity .~ 0.17
+    <> V.lens.layer.shadowColor .~ UIColor.ksr_dropShadow.cgColor
```

Comment:
This looks exactly like `dropShadowStyleMedium` except for the shadow color. Is there a reason the black color wasn't working? According to the style guide, there should only be the 3 types of shadow, all with black at 0.17 opacity. I think we should be updating the base `dropShadowStyle` to black with 0.17 opacity. Once this is updated, you can drop the lines for `shadowOpacity` and `shadowColor` in the medium and large styles.  Let me know if you need more clarification. cc @maggieKSR